src/components/TouristInfoCards.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+var TouristCardsLinksAndImages = [
+  {
+    CityName: "Glasgow",
+    Link: "https://peoplemakeglasgow.com/",
+    Photo: "https://peoplemakeglasgow.com/templates/people/images/logo.png"
+  },
```

Comment:
The usual naming convention for Javascript keys is to use camelcase but with the first letter being in lower case. Not a big deal, but it's more about being consistent with the JS convention: so `cityName` instead of `CityName` etc...

Reply:
uppercase again :D